test/OpenTelemetry.Tests/Metrics/AggregatorTest.cs

```diff
@@ -188,6 +188,40 @@ public void HistogramWithOnlySumCount()
             Assert.False(enumerator.MoveNext());
         }
 
+        [Fact]
+        public void MultiThreadedHistogramUpdateAndSnapShotTest()
```

Comment:
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs#L1629 What about the existing tests?